src/components/Equations.ts

```diff
@@ -109,6 +109,7 @@ export class Equations<TGameStartr extends FullScreenPokemon> extends Component<
         const ev: IValuePoints = this.newPokemonEVs();
         const iv: IValuePoints = this.newPokemonIVs();
 
+        this.gameStarter.modAttacher.fireEvent("onNewPokemonCreation", chosenInfo);
```

Comment:
Let's put a blank line between this and before the `return` so they're more visually distinct.